File: Models/PageModel/PageWidget/WidgetContent/VideoConfiguration.cs
using System;
using System.Collections.Generic;

namespace Model.PageModel.PageWidget.WidgetContent
{
    public class VideoConfiguration : BaseMediaConfiguration
    {
        public string Width { get; set; }
        public string Height { get; set; }
        public bool EnableLoop { get; set; }
        public bool EnableAutoplay { get; set; }
        public bool DisableControls { get; set; }
        public bool Responsive { get; set; }
    }
}

File: resources/Models/video-configuration.ts
import { BaseMediaConfiguration } from "./base-media-configuration";

export interface VideoConfiguration extends BaseMediaConfiguration {
  
  Width?: number | string;
  
  Height?: number | string;
  
  EnableLoop?: boolean;
  
  EnableAutoplay?: boolean;
  
  DisableControls?: boolean;

  Responsive?: boolean;

}
